lisa/tools/hyperv.py

```diff
@@ -156,6 +156,12 @@ def create_vm(
         if not is_ready:
             raise LisaException(f"VM {name} did not start")
 
+    def get_first_switch(self) -> str:
```

Comment:
1. It may need to check if the first listed external switch.
1. Because it's used like "default" switch, so it should be `get_default_switch` too.
1. It may need more information from switches, so we need to define an object for switch.

Reply:
Done all three. Since we are looking for external switch I named it `get_default_external_switch`.